generator/internal/rust/templates/crate/src/builder.rs.mustache

```diff
@@ -40,9 +40,9 @@ pub mod {{Codec.ModuleName}} {
     ///     .build().await?;
     /// # gax::Result::<()>::Ok(()) });
     /// ```
-    {{/Codec.SkipBuilderDocs}}
     pub type ClientBuilder =
         gax::client_builder::ClientBuilder<client::Factory, gaxi::options::Credentials>;
+    {{/Codec.HasVeneer}}
 
     pub(crate) mod client {
```

Comment:
Do we still use the code inside `pub(crate) mod client`?

Reply:
No, we don't. We use `Client::new(...)` directly. Removed.